api/src/actions/click-event.ts

```diff
@@ -15,6 +18,16 @@ export const getClickEvents = async (date?: Date) => {
     return events
 }
 
+export const paginatedGetClickEvents = async (afterDate: string, afterID: String, limit: number, projectName: String) => {
+    await dbConnect();
+    const project = await ProjectModel.findOne({ projectName: projectName })
+    if (project != null && project._id != null) {
+        let projectId = project._id;
+        const events = await ClickEventModel.find({ date: { $gte: afterDate }, _id: { $gte: afterID }, projectId: projectId }).limit(limit);
```

Comment:
Make a search query that accounts for afterId being null - an example of this would be here:

https://github.com/GTBitsOfGood/Healing4Heroes/blob/9f51596189f38e20b4cf6393ac3c3c5594b66b98/backend/server/mongodb/actions/User.ts#L95